web/src/engine/websites/MangaDistrict.ts

```diff
@@ -4,10 +4,14 @@ import { DecoratableMangaScraper } from '../providers/MangaPlugin';
 import * as Madara from './decorators/WordPressMadara';
 import * as Common from './decorators/Common';
 
+const script = `
+    const images = [...document.querySelectorAll('div.page-break img')];
+    images.map(image => image.dataset['wpfcOriginalSrc'] || image.src);
+`
 @Madara.MangaCSS(/^{origin}\/read-scan\/[^/]+\/$/, 'meta[property="og:title"]:not([content*="MANGA DISTRICT"])')
 @Madara.MangasMultiPageAJAX()
 @Madara.ChaptersSinglePageAJAXv2()
-@Madara.PagesSinglePageCSS()
+@Common.PagesSinglePageJS(script, 1000)
```

Comment:
Suggestion : While it works, we should avoid injecting javascript when we can.

Please consider using 
`@Common.PagesSinglePageCSS(query, PageExtractor)` and define a `PageExtractor `that return either image.dataset.wpfcOriginalSrc either image.src :)

Like in : https://github.com/manga-download/haruneko/blob/master/web/src/engine/websites/ComicOnlineFree.ts#L11